### PR TITLE
[14.0][MIG] account_check_printing_report_base

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Backport from V15

@ForgeFlow

### CODE REVIEW

File: account_check_printing_report_base/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
```
Comment (AaronHForgeFlow) [inline account_check_printing_report_base/readme/CONTRIBUTORS.rst]: can you change eficent -> forgeflow everywhere?

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Having this error in runbot when confirming a payment:

```
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/opt/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/opt/odoo/odoo/http.py", line 696, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/odoo/http.py", line 370, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/odoo/http.py", line 358, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/odoo/http.py", line 919, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/odoo/http.py", line 544, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/addons/web/controllers/main.py", line 1374, in call_button
    action = self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/addons/web/controllers/main.py", line 1362, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/odoo/api.py", line 399, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/odoo/api.py", line 386, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/mnt/data/odoo-addons-dir/account_check_printing_report_base/models/account_payment.py", line 51, in action_post
    lambda x: x.journal_id.check_print_auto
  File "/opt/odoo/odoo/models.py", line 5314, in filtered
    return self.browse([rec.id for rec in self if func(rec)])
  File "/opt/odoo/odoo/models.py", line 5314, in <listcomp>
    return self.browse([rec.id for rec in self if func(rec)])
  File "/mnt/data/odoo-addons-dir/account_check_printing_report_base/models/account_payment.py", line 52, in <lambda>
    and x.payment_method_line_id.code == "check_printing"
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/odoo/odoo/http.py", line 652, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/odoo/http.py", line 317, in _handle_exception
    raise exception.with_traceback(None) from new_cause
AttributeError: 'account.payment' object has no attribute 'payment_method_line_id'
```
This is the journal configuration:

File: account_check_printing_report_base/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (JasminSForgeFlow) [reply account_check_printing_report_base/readme/CONTRIBUTORS.rst]: Updated, Thanks :)

**Comment by JasminSForgeFlow** [comment]
> Having this error in runbot when confirming a payment:
> 
> ```
> Error:
> Odoo Server Error
> 
> Traceback (most recent call last):
>   File "/opt/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
>     result = request.dispatch()
>   File "/opt/odoo/odoo/http.py", line 696, in dispatch
>     result = self._call_function(**self.params)
>   File "/opt/odoo/odoo/http.py", line 370, in _call_function
>     return checked_call(self.db, *args, **kwargs)
>   File "/opt/odoo/odoo/service/model.py", line 94, in wrapper
>     return f(dbname, *args, **kwargs)
>   File "/opt/odoo/odoo/http.py", line 358, in checked_call
>     result = self.endpoint(*a, **kw)
>   File "/opt/odoo/odoo/http.py", line 919, in __call__
>     return self.method(*args, **kw)
>   File "/opt/odoo/odoo/http.py", line 544, in response_wrap
>     response = f(*args, **kw)
>   File "/opt/odoo/addons/web/controllers/main.py", line 1374, in call_button
>     action = self._call_kw(model, method, args, kwargs)
>   File "/opt/odoo/addons/web/controllers/main.py", line 1362, in _call_kw
>     return call_kw(request.env[model], method, args, kwargs)
>   File "/opt/odoo/odoo/api.py", line 399, in call_kw
>     result = _call_kw_multi(method, model, args, kwargs)
>   File "/opt/odoo/odoo/api.py", line 386, in _call_kw_multi
>     result = method(recs, *args, **kwargs)
>   File "/mnt/data/odoo-addons-dir/account_check_printing_report_base/models/account_payment.py", line 51, in action_post
>     lambda x: x.journal_id.check_print_auto
>   File "/opt/odoo/odoo/models.py", line 5314, in filtered
>     return self.browse([rec.id for rec in self if func(rec)])
>   File "/opt/odoo/odoo/models.py", line 5314, in <listcomp>
>     return self.browse([rec.id for rec in self if func(rec)])
>   File "/mnt/data/odoo-addons-dir/account_check_printing_report_base/models/account_payment.py", line 52, in <lambda>
>     and x.payment_method_line_id.code == "check_printing"
> Exception
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "/opt/odoo/odoo/http.py", line 652, in _handle_exception
>     return super(JsonRequest, self)._handle_exception(exception)
>   File "/opt/odoo/odoo/http.py", line 317, in _handle_exception
>     raise exception.with_traceback(None) from new_cause
> AttributeError: 'account.payment' object has no attribute 'payment_method_line_id'
> ```
> 
> This is the journal configuration:
> 
> 

This issue has been fixed.
Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM

**Pull request merged**